src/main/resources/db/changelog/changesets/changelog_20230825T115511Z.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="hediliabd" id="1252713657904-1">
+        <addColumn tableName="node">
+            <column name="stashed" type="boolean"/>
```

Comment:
You could default it to false so that older studies won't break, I got an error when deleting a preexisting node because stashed was null

Reply:
indeed, added